ccat.py

```diff
@@ -354,6 +402,234 @@ def append_configuration(self, answers):
         })
 
 
+class GCP(object):
+    def __init__(self, service_account_json_file_path=None, access_token=None):
+        self.configuration = {
+            'service_account_json_file_path': service_account_json_file_path,
+            'access_token': access_token,
+        }
+
+        self.data = {}
+
+    def get_menu_choices_registries(self, gcp_registries=[]):
+        choices = []
+        for registery in gcp_registries:
```

Comment:
registery --> registry

Also there are 3 cases of the same thing here: https://github.com/RhinoSecurityLabs/ccat/blob/master/modules/ecr__pull_repos/main.py